packages/token/tokens/color/base.json

```diff
@@ -2,13 +2,15 @@
   "color": {
     "base": {
       "gray": {
-        "light" : { "value": "#CCCCCC", "darkValue": "#EDEDED" },
-        "medium": { "value": "#999999" },
-        "dark"  : { "value": "#111111" }
+        "light": {
+          "value": "rgba(255, 255, 255, 0.45)"
```

Comment:
color is independent of opacity can't we hex here or color without opacity

Reply:
<img width="546" alt="Screenshot 2023-09-26 at 12 39 54 PM" src="https://github.com/SigNoz/signoz-components-library/assets/26198062/ce58136c-6b8c-42ea-9129-95f7c1dcf752">

There are some colour tokens with opacity included in the Figma.